cost-analyzer/charts/prometheus/templates/node-exporter-ocp-scc.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if and (.Capabilities.APIVersions.Has "security.openshift.io/v1/SecurityContextConstraints") (.Values.global.platforms.openshift.scc.nodeExporter) (.Values.nodeExporter.enabled) }}
```

Comment:
Nice catch here. If I'm understanding correctly, OCP users who want to run Node Exporter need this `SecurityContextConstraints` to define extra permissions?

Is this how it currently works in the OpenShift Helm Chart as well?

Reply:
The current fork doesn't offer this ability at all. I added it as a measure of convenience for users but disabled by default. Many users prefer to use their own SCCs because of corporate policy. They are free to do so, but one non-default SCC must apply. If it does not, the Pods will be unable to start because OCP does not ship with an SCC which allows these types of permissive Pods to run.